docker/release/aarch64/Dockerfile

```diff
@@ -0,0 +1 @@
+../x86_64/Dockerfile
```

Comment:
ehm, is it supported by dockerfiles?

Reply:
It is actually a symlink. Git stores symlinks in blobs and restores a symlink when it is pulled back to the filesystem